modules/linear_boltzmann_solvers/a_lbs_solver/source_functions/source_function.cc

```diff
@@ -112,7 +111,7 @@ SourceFunction::operator()(const LBSGroupset& groupset,
             if (apply_ags_scatter_src_)
               for (const auto& [_, gp, sigma_sm] : S_ell.Row(g))
                 if (gp < gs_i_ or gp > gs_f_)
-                  rhs += sigma_sm * phi[gp];
+                  rhs += rho * sigma_sm * phi_im[gp];
```

Comment:
if this really rho or the ratio of the new to the old rho? if it is rho, does it mean that sigma_sm is now a microXS ? also, when I see rho, I wrongly think of it as g/cc first, but it's atom/cc, right?

Reply:
The short answer is that sigma_sm is microscopic. As the PR description says, the internals of the code now treat cross sections as microscopic. If the user does nothing, the densities are all one, it is the same as using macroscopic cross sections. Right now, there is no precise form for rho, but it must be consistent with the microscopic cross sections. The main idea of this is that you could have a combination of microscopic cross sections based on relative isotopic composition and a number density derived from something like a material density .